frontend/assets/css/tailwind.css

```diff
@@ -173,3 +173,12 @@
     @apply opacity-50;
   }
 }
+
+@media (max-width: 640px) {
```

Comment:
Thanks @bharath637462 🙏 Could you transfer this to the component itself though? I'll direct you where the change should happen and what it should be :)